plugins/git/receive-app

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$(dirname $0)/../common/functions"
+
+APP="$1"; REV="$2"
+
+dokku git-build $APP $REV
```

Comment:
I actually think this should probably be refactored out of commands directly in the plugin. I don't see the need now to be able to run git-build on it's own.